package.json

```diff
@@ -45,7 +45,8 @@
     "react-dom": "^16.8.3",
     "react-i18next": "^10.0.0",
     "react-redux": "^5.0.7",
-    "react-select": "^2.1.1"
+    "react-select": "^2.1.1",
+    "uuid": "^3.3.2"
```

Comment:
I don't see this dependency being used in this PR. Can we remove it? Or is it used elsewhere?

Reply:
it's used in Search.js https://github.com/Wikia/react-common/pull/22/files#diff-29c98872cca9a2d080301634fa42a45bR6 but apparently disappeared during previous PRs